src/Command/SplitDb.php

```diff
@@ -0,0 +1,269 @@
+     *
+     * @var MagentoShell
+     */
+    private $magentoShell;
+
+    /**
+     * @param LoggerInterface $logger
+     * @param DbConfig $dbConfig
+     * @param DeployInterface $stageConfig
+     * @param ConfigMerger $configMerger
+     * @param RelationshipConnectionFactory $connectionDataFactory
+     * @param ConfigReader $configReader
+     * @param MaintenanceModeSwitcher $maintenanceModeSwitcher
+     * @param MagentoShell $magentoShell
+     */
+    public function __construct(
+        LoggerInterface $logger,
+        DbConfig $dbConfig,
+        DeployInterface $stageConfig,
+        ConfigMerger $configMerger,
+        RelationshipConnectionFactory $connectionDataFactory,
+        ConfigReader $configReader,
+        MaintenanceModeSwitcher $maintenanceModeSwitcher,
+        MagentoShell $magentoShell
+    ) {
+        $this->logger = $logger;
+        $this->dbConfig = $dbConfig;
+        $this->stageConfig = $stageConfig;
+        $this->configMerger = $configMerger;
+        $this->connectionDataFactory = $connectionDataFactory;
+        $this->configReader = $configReader;
+        $this->maintenanceMode = $maintenanceModeSwitcher;
+        $this->magentoShell = $magentoShell;
+        parent::__construct();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure()
+    {
+        $types = array_keys(self::TYPE_MAP);
+        $this->setName(self::NAME)
+            ->setDescription(
+                'Move sales and/or checkout quote related tables to a separate DB servers.'
+                . ' This procedure is not allowed if you do not have prepared DBs'
+            );
+        $this->addArgument(
+            self::ARGUMENT_TYPES,
+            InputArgument::IS_ARRAY,
+            sprintf(
+                'Space-separated list of table types for migrating to appropriate DB or omit to apply to all'
+                . ' table types (possible types: %s).',
+                implode(',', $types)
+            ),
+            $types
+        );
+
+        parent::configure();
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        try {
+            $types = $input->getArgument(self::ARGUMENT_TYPES);
+            $configFormEnvFile = $this->configReader->read() ?? [];
+            $envDbConfig = $this->getDatabaseConfig();
+
+            $breakExecution = false;
+            foreach ($types as $type) {
+                $connectionName = self::TYPE_MAP[$type][DbConfig::KEY_CONNECTION];
+                if (isset($configFormEnvFile['db']['connection'][$connectionName])) {
+                    $output->writeln(sprintf('Database for %s has been split.', $type));
+                    $breakExecution = true;
+                    continue;
+                }
+                if (!isset($envDbConfig['connection'][$connectionName])) {
+                    $output->writeln(sprintf('ERROR: there is not connections to additional DBs for %s splitting.', $type));
```

Comment:
I suggest to add link where customer can find more about splitting DB on Cloud how to retrieve/check this connection.

Reply:
Also, this line is more than 120 symbols. Please fix it and other 2 places